docs/get_started.md

```diff
@@ -13,7 +13,7 @@ Here we walk through how to quickly setup conflux full node to join the conflux
 
 ## Running a Conflux Node
 
-First, please build the binary as instructed in [Installation](install.md#Install).
+First, please [Get a Conflux Client](../../run-a-node/en/how-to-get.md) or [Compile from Source](install.md).
```

Comment:
This only works within `conflux-developer-site`. A user was confused because the "Running a Conflux Node" only linked to building from source, so I think it's better to change it like this.